src/genc.rs

```diff
@@ -8,10 +8,9 @@ use std::str::FromStr;
 pub enum Backend {
     Cuda,
     ISPC,
-    MulticoreC,
-    None,
```

Comment:
The `None` backend was kept here to make it possible to parse the headers and test it without the presence of the futhark binary so I do not think it should be removed.

Reply:
Actually disregard this comment